web/themes/joinup/templates/layout/page.html.twig

```diff
@@ -62,53 +62,57 @@
 
   {%
   set main_classes = [
-    'main-container',
-    'mdl-grid',
-    not page.sidebar_left ? 'main-container--center',
+    not page.sidebar_left ? 'mdl-grid--center',
   ]
   %}
-  <div class="section section--content">
-    <main class="{{ main_classes|join(' ') }}" role="main">
+  <main class="section section--content" role="main">
+    <div class="main-container">
       <a id="main-content" tabindex="-1"></a>{# link is in html.html.twig #}
+      <div class="row">
+        <div class="mdl-grid {{ main_classes|join(' ') }}">
+          {% if page.sidebar_left and not has_vertical_tabs %}
+            <aside class="section section--sidebar-left mdl-cell mdl-cell--2-col" role="complementary">
+              {{ page.sidebar_left }}
+            </aside>
+          {% endif %}
 
-      {% if page.sidebar_left and not has_vertical_tabs %}
-        <aside class="section section--sidebar-left mdl-cell mdl-cell--2-col" role="complementary">
-          {{ page.sidebar_left }}
-        </aside>
-      {% endif %}
+          <div class="section section--main mdl-cell {{ has_vertical_tabs ? 'mdl-cell--12-col mdl-grid--no-spacing content-container--form' : 'mdl-cell--8-col content-container' }}">
+            {% if page.content_top %}
+              <div class="section section--content-top {{ has_vertical_tabs ? 'mdl-cell mdl-cell--8-col mdl-cell--2-offset-desktop form__cell'}}">
```

Comment:
For the region check - [ISAICP-3660](https://webgate.ec.europa.eu/CITnet/jira/browse/ISAICP-3660)
For the offset - just please test whether the offset will be removed when there is no element in the check that needs the offset.